x-pack/plugin/security/src/test/java/org/elasticsearch/license/LicensingTests.java

```diff
@@ -245,7 +245,7 @@ private void disableLicensing() throws Exception {
         // wait for things to stabilize!
         assertBusy(() -> {
             for (XPackLicenseState licenseState : internalCluster().getInstances(XPackLicenseState.class)) {
-                if (licenseState.isAuthAllowed() == false) {
+                if (licenseState.isSecurityEnabled() == false) {
```

Comment:
We briefly chatted about this elsewhere but in this test class security should always be enabled so I think that this should no longer be in a loop over internal state but just go ahead and call `enableLicensing(OperationMode.BASIC)`